kubernetes-kit-starter/src/main/java/com/vaadin/kubernetes/starter/sessiontracker/SessionSerializationCallback.java

```diff
@@ -0,0 +1,20 @@
+package com.vaadin.kubernetes.starter.sessiontracker;
+
+/**
+ * Error callbacks that are called after a serialization or deserialization
+ * error happens.
+ */
+public interface SessionSerializationCallback {
+
+    /**
+     * Error callback that is called after a serialization error happens.
+     */
+    default void onSerializationError(Exception ex) {
+    }
+
+    /**
+     * Error callback that is called after a deserialization error happens.
+     */
```

Comment:
I would add a note indicating that the callback should not throw any exception.

Reply:
Updated.